sources/packages/api/src/route-controllers/student/student.controller.ts

```diff
@@ -501,4 +503,54 @@ export class StudentController extends BaseController {
       restrictionMessage: studentRestrictionStatus.restrictionMessage,
     } as StudentRestrictionDTO;
   }
+
+  /**
+   * API to fetch student details by studentId.
+   * This API will be used by ministry users.
+   * @param studentId
+   * @returns Student Details
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Get("/aest/:studentId")
```

Comment:
The URI here would be api/students/aest/123, it does not look like the ones that we have been creating.
I believe that AEST/Student/Institution would be always at the most starting as possible.
I would assume that the API for this case would be something like
```
api/aest/student/123
```
@ann-aot @guru-aot thoughts?

Reply:
Given that discussion is going to happen on organizing the APIs, I want to keep the APIs in existing controllers itself. The pattern of continuous numbers in API is now fixed.